src/workflows/steps/index.js

```diff
@@ -962,13 +967,20 @@ export function verifyUpstream(state) {
 		return repository
 			.getDetails()
 			.then(response => {
-				const parent_ssh_url = response.data.hasOwnProperty("parent")
-					? response.data.parent.ssh_url
-					: response.data.ssh_url;
+				let parent_ssh_url;
+				if (response.data.hasOwnProperty("parent")) {
+					parent_ssh_url = origin.url.includes("https")
+						? response.data.parent.svn_url
+						: response.data.parent.ssh_url;
+				} else {
+					parent_ssh_url = origin.url.includes("https")
+						? response.data.svn_url
+						: response.data.ssh_url;
+				}
 				const command = `git remote add upstream ${parent_ssh_url}`;
 				return util
 					.exec(command)
-					.then(util.log.end)
+					.then(util.log.end())
```

Comment:
Did this give you a problem? I see you added the parens back

Reply:
It seemed to not call it whenever I ran `tag-release` and it also made the tests fail until I added back the (). So, I'm not sure what is going on there. =(